test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void names_areEqual() throws IllegalValueException {
+	Name personOne = new Name("John Doe");
```

Comment:
I think you can refactor this code into another method, accepting two strings as the argument.